docs/install.md

```diff
@@ -80,4 +93,7 @@ Drupal Compatibility
     <tr>
         <td>✓</td> <td>Compatible but no longer supported</td>
     </tr>
+    
 </table>
+
+❶: EOL date for Drush 8 tbd, but estimated to be in concert with <a href="https://www.drupal.org/psa-2019-02-25">Drupal 7 EOL</a>.</td>
```

Comment:
maybe spell out TBD for non english as first language folks? At least capitalize for clarity its an acronym.
```suggestion
❶: EOL date for Drush 8 TBD, but estimated to be in concert with <a href="https://www.drupal.org/psa-2019-02-25">Drupal 7 EOL</a>.</td>
```